packages/checkout/widgets-lib/src/widgets/sale/hooks/useCurrency.ts

```diff
@@ -0,0 +1,58 @@
+import { Environment } from '@imtbl/config';
+import { useState, useCallback } from 'react';
+
+type CurrencyResponse = {
+  decimals: number;
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  erc20_address: string;
+  name: string;
+};
+
+const PRIMARY_SALES_API_BASE_URL = {
```

Comment:
This is already defined in `useSignOrder`

Reply:
@mimi-imtbl lets move this up to utils/config